src/main/java/com/google/graphgeckos/dashboard/datatypes/Log.java

```diff
@@ -23,10 +24,17 @@
 public class Log {
 
   // Log type (e.g "stdio")
-  private final String type;
+  @Field(name = "type")
+  private String type;
 
   // Log link (e.g "http://lab.llvm.org:8011/builders/mlir-nvidia/builds/6403/logs/stdio")
-  private final String link;
+  @Field(name = "link")
+  private String link;
+
+  /**
+   * Used only by Spring GCP.
+   */
+  public Log() { }
```

Comment:
supernit: don't use a space in an empty block: https://google.github.io/styleguide/javaguide.html#s4.1.3-braces-empty-blocks
```suggestion
  public Log() {}
```